kotlin-antd/src/main/kotlin/antd/menu/MenuItem.kt

```diff
@@ -18,6 +18,10 @@ external interface MenuItemProps : RProps {
     var onClick: ((param: ClickParam) -> Unit)?
     var onMouseEnter: ((e: ItemClickParam) -> Unit)?
     var onMouseLeave: ((e: ItemClickParam) -> Unit)?
+    var key: Any? /* Object */
+    var itemIcon: Any? /* ReactElement | ((p: MenuItemProps) -> ReactElement */
+    var icon: Any? /* ReactElement */
```

Comment:
why not make this
`var icon: ReactElement?`
ReactElement is a supported type

Reply:
Yep, it's fine, in the Button component it's like this right now